pandas/io/formats/excel.py

```diff
@@ -299,6 +299,24 @@ def _border_style(self, style: str | None, width: str | None, color: str | None)
             if width_name in ("hair", "thin"):
                 return "dashed"
             return "mediumDashed"
+        elif style in (
+            "hair",
+            "mediumDashDot",
```

Comment:
Is it easy to test the rest of these options?

Reply:
Thanks for the suggestion, it turned up an issue due to case lowering. Fix is in place now.